application/deed/views.py

```diff
@@ -29,6 +30,26 @@ def get_deed(deed_reference):
     return jsonify({"deed": result.deed}), status.HTTP_200_OK
 
 
+@deed_bp.route('', methods=['GET'])
+def get_deeds_status_with_mdref_and_title_number():
+    md_ref = request.args.get("md_ref")
+    title_number = request.args.get("title_number")
+
+    if md_ref and title_number:
+        deeds_status = Deed.get_deed_status(title_number, md_ref)
+
+        if len(deeds_status) == 0:
+            abort(status.HTTP_404_NOT_FOUND)
+
+        return Response(
+            json.dumps(deeds_status),
+            status=200,
+            mimetype='application/json'
+        )
```

Comment:
jsonify as part of flask used to have a security feature that didnt allow you to dump arrays that look like 

```
[
   {
      "stuff" : "stuff"
   },
   {
      "stuff" : "stuff"
   },
   {
      "stuff" : "stuff"
   }
]
```
because their was a flaw in some older browsers, those browsers have since been patched and jsonify in the flask repo now allows it however it has not been tagged for a new version since so we cant pull in the change as its not stable.